.github/workflows/security.yaml

```
@@ -0,0 +1,32 @@
+---
+name: security
+
+on:
+  push:
+    branches: [main]
+  release:
+    types: [published]
+  pull_request:
+    branches: [main]
+  schedule:
+    - cron: '0 8 * * 1' # Monday - 8am - UTC
+  workflow_dispatch:
+
+jobs:
+  build:
+    permissions:
+      contents: read
+      packages: write
+    uses: ./.github/workflows/docker-build.yaml
```

Just noticed another error raised by the workflow.
```suggestion
    uses: ./.github/workflows/docker-build.yml
```